packages/plugin-core/src/survey.ts

```diff
@@ -118,6 +118,8 @@ export class BackgroundSurvey extends DendronQuickPickSurvey {
       });
     }
 
+    const role = maybeOtherResult || result.label;
+    AnalyticsUtils.identify({ role });
```

Comment:
if we call identify here with the role and in the future call identify without it, just checking that future calls without the role won't reset the property?

Reply:
yes. once a user property is set, it will persist. they will then be included with every event captured until another identify call happens, at which point any number of user properties can update.

Reply:
I'll DM you a user lookup to verify.

Reply:
Do you mean by that once we get an identify with `role` set once, we can set the cohort to "users who identified with as this role ever?"

If that's the case, looking at amplitude, it looks like you can at most ask for users who had a property within the last 90 days. We might need to save the role in meta, and keep identifying the user with that role.

Reply:
> Do you mean by that once we get an identify with role set once, we can set the cohort to "users who identified with as this role ever?" 
yes

> (I'm assuming that the role will be unset once the user restarts VSCode)
nope, not setting a property persists the property